config/prow/jobs/custom/test-infra.yaml

```diff
@@ -102,7 +102,29 @@ presubmits:
       - name: test-account
         secret:
           secretName: test-account
-
+  knative/community:
+  - name: pull-knativeteam-groups-test
+    annotations:
```

Comment:
We normally don't show presubmit run results on TestGrid since they are mostly noisy.

Reply:
i'll get rid of this